### PR TITLE
:^ ) face on mediborg

### PR DESCRIPTION
**Pull request by Super3222**
### Intent of your Pull Request
#746

Mediborgs can now choose from a standard mediborg icon to a mediborg icon that has a :^ ) face on it.
http://imgur.com/y9EORZm

### CODE REVIEW

**Comment by ShadowDeath6** [comment]
noice

File: code/modules/mob/living/silicon/robot/robot.dm [hunk 1]
```diff
@@ -207,12 +207,20 @@
...
 			module = new /obj/item/weapon/robot_module/medical(src)
 			hands.icon_state = "medical"
-			icon_state = "mediborg"
-			animation_length = 35
+			switch(icontype)
+				if("Medborg")
+					icon_state = "mediborg"
+					animation_length = 35
+				if("Succ Baby")
+					icon_state = "mediborg+succ"
```
Comment (Cruix) [inline code/modules/mob/living/silicon/robot/robot.dm]: Please give it a serious name

**Comment by FluffySurvivor** [comment]
:^)

**Comment by ShadowDeath6** [comment]
Fix yer fukkn conflicts >:^(

**Comment by AsV9** [comment]
this is the silliest thing..

**Pull request closed without merging**